src/schema-manager/file-schema-manager.test.ts

```diff
@@ -1,5 +1,5 @@
 import { GraphQLSchema, graphql, buildSchema } from 'graphql';
-import { introspectionQuery } from 'graphql/utilities';
+import { getIntrospectionQuery } from 'graphql/utilities';
```

Comment:
Thx.  I think this change of the function name does not backward compatibility. So I will release ts-graphql-plugin@v2.0.0, which requires `graphql >= 15.x` .